src/components/transport_manager/include/transport_manager/websocket_server/websocket_secure_session.h

```diff
@@ -45,6 +45,7 @@ class WebSocketSecureSession : public WebSocketSession<ExecutorType> {
   WebSocketSecureSession(tcp::socket,
                          ssl::context& ctx,
                          DataReceiveCallback data_receive,
+                         DataReceiveCallback on_data_sent,
```

Comment:
@mvorobio 
Strange looking name the  DataReceiveCallback for the  on_data_sent

Reply:
@SNiukalov  please check https://github.com/LuxoftSDL/sdl_core/pull/18/commits/7bc1f2ce4c04d80778a833d7022995d806ce5788